manifests/pulp.pp

```diff
@@ -57,9 +58,18 @@
 
   contain ::pulp
 
-  foreman::config::passenger::fragment { 'pulp':
-    content     => file('katello/pulp-apache.conf'),
-    ssl_content => file('katello/pulp-apache-ssl.conf'),
+  unless $manage_httpd {
+    foreman::config::passenger::fragment { 'pulp':
+      content     => file('katello/pulp-apache.conf'),
+      ssl_content => file('katello/pulp-apache-ssl.conf'),
+    }
+
+    file { $repo_export_dir:
```

Comment:
Why move this? I think people export content regardless of Apache. Some people export onto external media and import into another Katello server, I know others who export it so its available via HTTP and change sync across the network. https://theforeman.org/plugins/katello/3.4/user_guide/disconnected/index.html

Reply:
This is an ugly hack that allowed me to move forward. The problem is that it tries to create this directory owned by ```foreman:foreman``` which doesn't exist if you only install pulp on a node. See https://groups.google.com/forum/#!topic/foreman-dev/sofiVxnbLGQ for some background.